src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/app/nav-menu/nav-menu.component.html

```diff
@@ -7,6 +7,9 @@
         <span class="navbar-toggler-icon"></span>
       </button>
       <div class="navbar-collapse collapse d-sm-inline-flex flex-sm-row-reverse" [ngClass]='{"show": isExpanded}'>
+        <!--#if (IndividualLocalAuth) -->
+        <app-login-menu></app-login-menu>
```

Comment:
It's surprising that `<app-login-menu></app-login-menu>` appears *before* the links to Home/Counter/etc in the source, given that it appears *after* them in the UI. Is this a workaround for a CSS issue? Could the underlying CSS issue be fixed instead?

Reply:
Likely due to 'flex-sm-row-reverse' I think we just stack things right to left